src/ontology/mondo-ingest.Makefile

```diff
@@ -413,19 +413,15 @@ tmp/merged.owl: tmp/mondo.owl mondo-ingest.owl tmp/mondo.sssom.ttl
 	$(ROBOT) merge -i tmp/mondo.owl -i mondo-ingest.owl -i tmp/mondo.sssom.ttl --add-prefixes config/context.json \
 			 remove --term "http://purl.obolibrary.org/obo/mondo#ABBREVIATION" --preserve-structure false -o $@
 
-tmp/merged.db: tmp/merged.owl
+tmp/%.db: tmp/%.owl
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	@rm -f tmp/merged-relation-graph.tsv.gz
-	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
+	RUST_BACKTRACE=full semsql make tmp/$*.db -P config/prefixes.csv
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	@rm -f tmp/merged-relation-graph.tsv.gz
 
-mondo-ingest.db: tmp/merged.db
```

Comment:
Is it a good idea to get rid of this goal? What if someone else depends on this?

Reply:
<details><summary>Details</summary>
<p>

Ah, that has been replaced by `tmp/%.db`.

Right now only this PR depends on this. But strangely, I just noticed it is in the middle of the `build-mondo-ingest` command. I would imagine it should be removed from there?

```make
build-mondo-ingest:
	$(MAKE) refresh-imports exclusions-all mondo-ingest.db slurp-all mappings matches \
		mapped-deprecated-terms mapping-progress-report \
		recreate-unmapped-components sync documentation
```

</p>
</details>

Reply:
Not sure, if it is listed here, maybe it serves a purpose? How does removing this help this PR?

Reply:
@matentzn I moved these changes for you into:
- #374

Reply:
- [ ] Drop the one that goes into the root